partiql-plan/src/main/kotlin/org/partiql/plan/v1/rel/RelUnion.kt

```diff
@@ -0,0 +1,67 @@
+package org.partiql.plan.v1.rel
+
+/**
+ * Logical `UNION [ALL|DISTINCT]` operator for set (or multiset) union.
+ */
+public interface RelUnion : Rel {
+
+    public fun isAll(): Boolean
```

Comment:
nit: wonder if it would be preferred to keep the set quantifier naming consistent. In `RelUnion`, `RelExcept` and `RelIntersect`, we use `isAll` while in `RelAggregateCall` we use `isDistinct`.

Reply:
This has to do with which is the "default".

For union/intersect/except, DISTINCT is default — so "isAll()" is like asking "is this different than the default"?

For aggregate calls, ALL is default — so "isDistinct()" is like asking "is this different than the default"?